README.md

```diff
@@ -31,7 +45,7 @@ Ways to get the `cabal-install` binary
 2. _[Download from official website](https://www.haskell.org/cabal/download.html)_:
     the `cabal-install` binary download for your platform should contain the `cabal` executable.
 
-#### Preview Releases
+### Preview Releases
 
 _Getting unreleased versions of `cabal-install`_: gives you a chance to try out yet-unreleased features.
```

Comment:
```suggestion
Getting unreleased versions of `cabal-install` gives you a chance to try out yet-unreleased features.
```

The colon and italics don't make sense after turning this from a bullet point to a regular paragraph.